src/coreclr/src/jit/compiler.h

```diff
@@ -8154,8 +8166,9 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
         return emitTypeSize(TYP_SIMD8);
     }
 
+public:
     // Returns the codegen type for a given SIMD size.
-    var_types getSIMDTypeForSize(unsigned size)
+    static var_types getSIMDTypeForSize(unsigned size)
```

Comment:
We could also change `GenTreeHWIntrinsic` to track the SimdType rather than the SimdSize, but it is a more involved change.